spec/features/guider_views_appointments_spec.rb

```diff
@@ -42,6 +42,78 @@
     end
   end
 
+  context 'Appointment is updated by someone else', js: true do
+    scenario 'viewing their appointments' do
+      @guider = create(:guider)
```

Comment:
Why not extract these factory calls and assignments to something step-like (`given_the_requisite_background_data` or something) that is shared between this and the following scenario?